runelite-client/src/main/java/net/runelite/client/plugins/wintertodt/WintertodtPlugin.java

```diff
@@ -182,6 +184,24 @@ public void onGameTick(GameTick gameTick)
 		checkActionTimeout();
 	}
 
+	@Subscribe
+	public void onVarbitChanged(VarbitChanged varbitChanged)
+	{
+		if (config.notifyWhenRoundStarting())
+		{
+			int currentTimerValue = client.getVar(VarPlayer.WINTERTODT_TIMER);
+			if (currentTimerValue > 0)
+			{
+				log.debug("Wintertodt time left until round start " + currentTimerValue);
+
+				if (currentTimerValue < 50000 && currentTimerValue != 0)
+				{
+					notifier.notify("Round is about to start");
```

Comment:
This seems like it would cause notifications to be spammed whenever the time was low

Reply:
I've used this notification from 70-97 now and it notifies at max two times

Reply:
That may just be your system notifications being smart. You can run the client in debug and see notifier logs.

Reply:
That makes sense, I will check this tonight and add a debounce if needed